courses/free-courses-te.md

```diff
@@ -0,0 +1,206 @@
+# Index
+
+* [Android](#android)
+* [Angular](#angular)
+* [Artificial Intelligence](#artificial-intelligence)
+* [Automata Theory](#automata-theory)
+* [Bash and Shell](#bash-and-shell)
+* [C](#c)
+* [C++](#c++)
+* [Cloud Computing](#cloud-computing)
+* [Compiler Design](#compiler-design)
+* [CSS](#css) 
+* [Database Management Systems](#database-management-systems)
+* [Data Science](#data-science)
+* [Data Structures](#data-structures)
+* [Deep Learning](#deep-learning)
+* [Ethical Hacking](#ethical-hacking)
+* [ExpressJS](#expressjs)
+* [HTML](#html)
+* [Java](#java)
+* [Javascript](#javascript)
+* [Laravel](#laravel)
+* [Machine Learning](#machine-learning)
+* [MongoDB](#mongodb)
+* [NextJS](#nextjs)
+* [NodeJS](#nodejs)
+* [PHP](#php)
+* [Python](#python)
+* [ReactJS](#reactjs)
+* [SQL](#sql)
+* [VueJs](#vuejs)
+
+
+## NPTEL
```

Comment:
It is a site where courses can be done. Apart from that, reading material is provided. Yeah, I should have added it to the index. I guess it can be used both in the books file and here.